src/component/list.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import "./../style/list.scss";
+
+const List = (props) => {
+    const taskItems = props.tasks.map((task, index) => {
+        return <li key={index}>{task}</li>;
+    });
+
+    return (
+        <div id="taskList">
+            <h2>Tasks to do:</h2>
+            {
+                props.tasks.length > 0 ?
+                    <ul>
+                        {taskItems}
+                    </ul>
+                    :
+                    <p>You currently have no tasks to complete.</p>}
```

Comment:
Move this ending curly bracket to the next line to increase readability

Reply:
Dealt with in https://github.com/SaileshPatel/productivity-app-react-custom-build/pull/3/commits/4fab87bfba1a08375eaa3f490ad15da6a647b6f8